plugin-redfish/rfphandler/redfishEvents.go

```diff
@@ -44,7 +45,7 @@ func RedfishEvents(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
-
+	log.Println("Event Request", req)
```

Comment:
would this printed value be human readable?

Reply:
It will be print like this,

time="2021-02-17T11:07:09Z" level=info msg="Event Request map[@odata.context:/redfish/v1/$metadata#Event.Event @odata.type:#Event.v1_0_0.Event Events:[map[EventId:e73d1ba3-dfae-2f52-1170-5e0d6556da0f EventTimestamp:2021-02-17T10:50:52Z EventType:Alert MemberId:0 MessageArgs:[Lit] MessageId:iLOEvents.2.2.IndicatorLEDStateChanged Oem:map[Hpe:map[@odata.context:/redfish/v1/$metadata#HpeEvent.HpeEvent @odata.type:#HpeEvent.v2_1_0.HpeEvent Resource:/redfish/v1/Systems/1/]] OriginOfCondition:/redfish/v1/Systems/1/ Severity:OK]] Name:Events]"